tests/test_wheel.py

```diff
@@ -51,7 +51,9 @@
 platform_ = sysconfig.get_platform()
 if platform.system() == 'Darwin':
     parts = platform_.split('-')
-    parts[1] = platform.mac_ver()[0]
+    parts[1] = platform.mac_ver()[0].split('.')[0]
+    if parts[1] in ('11', '12'):
```

Comment:
Make it `> '11'`, to avoid this breaking as soon as macOS 13 comes out?

Reply:
The idea is that `packaging` would have this fixed by then.

Reply:
Even with that fix, it still needs to be `13.0` for any 13.x OS version in the absence of `MACOSX_DEPLOYMENT_PLATFORM`. The `packaging` fix is only for when `MACOSX_DEPLOYMENT_PLATFORM` is explicitly set.